### PR TITLE
update icon: kubernetes (original, original-wordmark, plain, plain-wordmark, line, line-wordmark)

### PR DESCRIPTION
**Pull request by canaleal**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR is related to #1998**

## Link to prove your SVG is correct and up-to-date.

- [Kubernetes Website](https://kubernetes.io/)

## Additional Notes

The ship wheel was clipped out from the plain and line SVGs. I did this because the ship wheel is ontop the blue heptagon as a white path. 

The blue color `#326ce5` was used for the font SVGs.

### CODE REVIEW

**Review by lunatic-fox — APPROVED** [review]
Everything looks fine! 👍🏼
Thank you for this contribution! 🚀
image
image
image
image

File: icons/kubernetes/kubernetes-plain-wordmark.svg
Comment (weh) [inline icons/kubernetes/kubernetes-plain-wordmark.svg:1]: this looks like it is not vertically centered.
The rest ist good :+1:

**Review by weh — CHANGES_REQUESTED** [review]
Just one image looks like it is not centered (vertical)
The rest is good.

File: icons/kubernetes/kubernetes-original-wordmark.svg
Comment (Snailedlt) [inline icons/kubernetes/kubernetes-original-wordmark.svg:1]: This needs to be vertically centered too

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Like @weh already mentioned the wordmark versions appear to not be vertically centered

**Review by weh — APPROVED** [review]
Top 👍

**Review by lunatic-fox — APPROVED** [review]
💡 It appears to not be vertically centered, however if you zoom in you can see that it's properly centered. Here's an example:
image
image

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**